subprojects/docs/src/docs/release/notes.md

```diff
@@ -54,6 +54,35 @@ The allowed labels are:
 
 More details can be found in the [Gradle Wrapper](userguide/gradle_wrapper.html#sec:adding_wrapper) section.
 
+### Kotlin DSL
+
+#### Experimental Property assignment for Kotlin scripts
+
+It's now possible to use experimental Property assignment for Kotlin scripts.
+This reduces the verbosity of Kotlin DSL when recommended [lazy property types](userguide/lazy_configuration.html#lazy_properties) are used in tasks and extensions.
+
+```kotlin
+interface Extension {
+    val description: Property<String>
+}
+
+extension {
+    // Old way, with `.set()` method call (still supported)
+    description.set("Hello Property")
+    // Gradle 8.1+ with Property assignment enabled
+    description = "Hello Property"
+}
+```
+
+You can enable Kotlin assignment by adding `systemProp.org.gradle.unsafe.kotlin.assignment=true` to the `gradle.properties` file.
+For IntelliJ IDEA support that property has to be set in the project root `gradle.properties` file (also for all included builds).
+Additionally, when the property's value is changed, the project has to be reopened in IntelliJ.
+Property assignment is supported from IntelliJ 2022.3 and from Android Studio Giraffe.
```

Comment:
🤔 Do we need to mention there are bugs?

Reply:
Isn't that a bit harsh? You know better than anyone how useful the IDE support is at the moment, so it's up to you, but we shouldn't overpromise (nor underpromise) on the expected experience.

Reply:
Removed that sentence. I think it should be good now.